docs/content/doc/developers/guidelines-frontend.md

```diff
@@ -40,25 +40,30 @@ We recommend [Google HTML/CSS Style Guide](https://google.github.io/styleguide/h
 1. Every feature (Fomantic-UI/jQuery module) should be put in separate files/directories.
 2. HTML ids and classes should use kebab-case.
 3. HTML ids and classes used in JavaScript should be unique for the whole project, and should contain 2-3 feature related keywords. We recommend to use the `js-` prefix for classes that are only used in JavaScript.
-4. jQuery events across different features should use their own namespaces.
-5. CSS styling for classes provided by frameworks should not be overwritten. Always use new class-names to overwrite framework styles. We recommend to use the `us-` prefix for user defined styles.  
+4. jQuery events across different features could use their own namespaces if there are potential conflicts.
+5. CSS styling for classes provided by frameworks should not be overwritten. Always use new class-names with 2-3 feature related keywords to overwrite framework styles.  
```

Comment:
I noticed that `us-` appear to be rarely used.
However, the added classes are hard to tell apart from Fomantic ones.
That could lead to anxiety to not remove classes because it is not clear, whether they are used elsewhere…

Reply:
I wrote the `us-` before but I found that it's very difficult to follow the rules strictly, and this rule has never been put into practice.

If a rule can not be followed, then maybe it's not worth to be written here. Indeed I do not want to make too much pressures for developers .....